src/app/public/modules/datepicker/datepicker-calendar.component.spec.ts

```diff
@@ -128,6 +128,11 @@ describe('datepicker calendar', () => {
     component = fixture.componentInstance;
   });
 
+  it('should be visible', () => {
+    fixture.detectChanges();
+    expect(component.datepicker.isVisible).toBeTruthy();
```

Comment:
Since `isVisible` and the other similar properties are all typed `boolean`, I'd prefer you use the more explicit `.toBe(true)`/`.toBe(false)` rather than `toBeTruthy()`/`toBeFalsy()`. That way we can be more confident that we're returning explicit `true`/`false` values for these properties at runtime instead of simply relying on the property types to be enforced it at build time.